src/services/authenticationService/authentications/createJWTAuthentication.ts

```diff
@@ -11,7 +11,7 @@ export function createJWTAuthentication(
   const { method, url } = requestData;
 
   const now = Math.floor(Date.now() / 1000);
-  const expire = now + 180;
+  const expire = now + (authenticationData.expiryTimeSeconds === undefined ? 180 : authenticationData.expiryTimeSeconds);
```

Comment:
I guess I could also use `??` -- do you prefer that?

Reply:
Yes, please